### ISSUE TITLE
Use with storybook not working

### ISSUE THREAD

**Issue by ssomnoremac** [robinpowered/glamorous-native#94]
- `glamorous-native` version: 1.4
- `react-native` version: 55.3
- `react` version: 16.3.1

What you did:

tried to use glamorous-native in storybook component files

What happened:

`./node_modules/react-native/Libraries/react-native/react-native-implementation.js`
`Module not found: Error: Can't resolve` any of the RN modules

**Comment by ajwhite** [comment]
Do you have a reproducible example you can share? One thing that strikes me in this bug report is that `react-native-implementation` is mentioned to not be found, however `glamorous-native` does not make use of this.

I'm curious if there's an unrelated factor at play here

**Comment by ssomnoremac** [comment]
I followed [Kurt's guide](https://hackernoon.com/up-and-running-with-universal-components-66678132cad) to set this up. The non-glamorous `Buttons` work fine. [Notes](https://github.com/ssomnoremac/UC/blob/master/packages/universal-components/components/notes/Card.js), however, do not.



Thanks!

**Comment by ssomnoremac** [comment]
```
ERROR in ./node_modules/react-native/Libraries/react-native/react-native-implementation.js
Module not found: Error: Can't resolve 'AccessibilityInfo' in '/Users/cameronmoss/Workspace/UC/packages/universal-components/node_modules/react-native/Libraries/react-native'
 @ ./node_modules/react-native/Libraries/react-native/react-native-implementation.js 19:35-63
 @ ./node_modules/glamorous-native/src/react-native-elements.js
 @ ./node_modules/glamorous-native/src/index.js
```

seems like it uses rn-elements which uses it

**Comment by ssomnoremac** [comment]
actually, it seems this is a webpack thing
```
index.js:16 Error: Module parse failed: Unexpected token (35:14)
You may need an appropriate loader to handle this file type.
| 
|       class GlamorousComponent extends React.Component {
|         state = {theme: null}
|         setTheme = theme => this.setState({theme})
| 
```

**Comment by ssomnoremac** [comment]
This is interesting. It's actually an issue with [react-native-web](https://github.com/necolas/react-native-web) that is being used to make universal components for storybook. What Kurt does is alias references of `react-native` with `react-native-web` like so 

.babelrc
```
"plugins": [
    "transform-class-properties",
    "transform-es2015-modules-commonjs",
    [
      "module-resolver",
      {
        "alias": {
          "react-native": "react-native-web"
        },
        "extensions": ["web.js", ".js"]
      }
    ]
```

To use glamorous it seems I'd need to compile it with a similar alias; however, a couple references from glamorous aren't there in `react-native-web` both relating to propTypes :(

```
Could not resolve "react-native-web/Libraries/Components/View/ViewStylePropTypes" in file /Users/cameronmoss/Workspace/glamorous-native/src/is-style-prop.js.
Could not resolve "react-native-web/Libraries/Text/TextStylePropTypes" in file /Users/cameronmoss/Workspace/glamorous-native/src/is-style-prop.js.
```

Any suggestions?

**Comment by ajwhite** [comment]
Ah, yes this is a known one. You can find more details about this in https://github.com/robinpowered/glamorous-native/issues/22. We've done a little brainstorming on the solution here

**Comment by ssomnoremac** [comment]
I hacked something together by patching the propTypes imports, upgrading all to react16 and getting rid of any `__ENV__` reference. I will try and work on that other issue. How should we close this/ handle usage in storybook with nr-web? Maybe a different branch?

**Comment by ajwhite** [comment]
The underlying issue isn't so much storybook, but supporting `react-native-web` detailed in #22. I'd recommend closing this as a fix for #22 inherently opens the door for using `react-native-web` with Storybook
